sql/core/src/test/scala/org/apache/spark/sql/execution/debug/DebuggingSuite.scala

```diff
@@ -48,4 +50,49 @@ class DebuggingSuite extends SparkFunSuite with SharedSQLContext {
     assert(res.forall{ case (subtree, code) =>
       subtree.contains("Range") && code.contains("Object[]")})
   }
+
+  test("SPARK-28537: DebugExec cannot debug broadcast or columnar related queries") {
```

Comment:
IMO this is of improvements for debugging, so we don't need the prefix. cc: @dongjoon-hyun

Reply:
`or` -> `and` in the title.

Reply:
It seems that @sarutak reported this issue as a `BUG`.

Reply:
I think this is a BUG.